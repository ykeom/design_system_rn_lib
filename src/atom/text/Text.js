import React, { Component } from 'react';
import { View, StyleSheet, Text as RNText } from 'react-native';

const styles = StyleSheet.create({
  root: {},
});

class Text extends Component {
  render() {
    return (
      <View style={styles.root}>
        <RNText>button</RNText>
      </View>
    );
  }
}

export default Text;
