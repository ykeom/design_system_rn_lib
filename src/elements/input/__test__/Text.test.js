import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Text from '../Text';

describe('Text', () => {
  it('should render', () => {
    const component = shallow(<Text />);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
