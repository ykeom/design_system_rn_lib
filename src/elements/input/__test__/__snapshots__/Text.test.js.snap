// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text should render 1`] = `
<View
  style={Object {}}
>
  <Text>
    text input
  </Text>
</View>
`;
